### PR TITLE
Replace RestClient by Net::HTTP and remove useless gems

### PR DESCRIPTION
**Pull request by vyper**
Resolves:
- #12 Remove rest-client
- #10 Remove useless gems

### CODE REVIEW

**Comment by vyper** [comment]
:shipit: @marcocarvalho @adilsoncarvalho @tiagopog (:

File: locasms.gemspec [hunk 1]
```diff
@@ -19,18 +19,12 @@ Gem::Specification.new do |spec|
...
 
   # test stuff
-  spec.add_development_dependency 'rspec', '~> 3.2'
-  spec.add_development_dependency 'fuubar', '~> 2.0.0'
-  spec.add_development_dependency 'timecop', '~> 0.7.3'
-
-  # run tests automatically
-  spec.add_development_dependency 'guard'
-  spec.add_development_dependency 'guard-rspec'
-  spec.add_development_dependency 'growl'
```
Comment (tiagopog) [inline locasms.gemspec]: Awesome! Less is more :clap:

File: spec/lib/locasms/client_spec.rb [hunk 2]
```diff
@@ -14,7 +14,7 @@
       end
     end
 
-    context 'When default' do
+    context 'When shortcode' do
```
Comment (tiagopog) [inline spec/lib/locasms/client_spec.rb]: Thanks for the fix :-)

**Comment by tiagopog** [comment]
:+1:

**Comment by adilsoncarvalho** [comment]
:+1:

**Comment by marcocarvalho** [comment]
:+1:

**Pull request merged**